app/views/articles/_actions.html.erb

```diff
@@ -47,7 +47,7 @@
             <button for="subscribe_notifications" id="notification-subscription-label_only_author_comments" class="notification-subscription-label" data-payload="only_author_comments">
               Post Author Comments <span class="selected-emoji">✅</span>
             </button>
-            <%= javascript_pack_tag "notificationSubscriptionHandler", defer: true %>
+            <%= javascript_packs_with_chunks_tag "notificationSubscriptionHandler", defer: true %>
```

Comment:
`javascript_packs_with_chunks_tag` is used instead so that we can benefit from code splitting.